pages/blog/index.vue

```diff
@@ -8,45 +8,67 @@
           </NuxtLink>
         </Center>
       </UContainer>
-      <UContainer class="py-10" :ui="{'constrained': 'max-w-2xl'}">
-        <h1>Neues aus der digitalen Welt</h1>
-        <p class="lead">Hier möchten wir gerne unser Wissen, über die digitale Zukunft, Technologien, Design und das Leben in einer digitalen Agentur, mit euch teilen, bleibt gespannt, wir sind es auch.</p>
+      <UContainer class="pt-16" :ui="{'constrained': 'max-w-7xl'}">
+        <Headline type="h2" class="pb-8 leading-[57px] text-[48px] lowercase">
```

Comment:
Please do not use [57px] and [48px] as discussed ready-made values that come close to that from Tailwind.